rust/src/lib/route.rs

```diff
@@ -14,6 +14,9 @@ use crate::{
 const DEFAULT_TABLE_ID: u32 = 254; // main route table ID
 const LOOPBACK_IFACE_NAME: &str = "lo";
 
+// TCP metric lock flags
+pub const TCP_METRICS_LOCK_CWND: u32 = 1 << 7;
```

Comment:
Please change 7 to `RTAX_CWND`, so future developer know what kernel constant to look for.